config.py

```diff
@@ -0,0 +1,10 @@
+#--------------------------------------------------------------------------------
+# These tokens are needed for user authentication.
+# Credentials can be generates via Twitter's Application Management:
+#       https://apps.twitter.com/app/new
+#--------------------------------------------------------------------------------
+
+consumer_key = "XxXxXxxXXXxxxxXXXxXX"
```

Comment:
No... I haven't thought about it for this context. I figured for now, every dev can just use their own Twitter dev account. At some point, we'll need to get one for the @presterity user, and the production scripts will auth as that user. Or maybe a @presterity-dev user.